### ISSUE TITLE
feat: sort list of scope dropdown values

### ISSUE THREAD

**Issue by WuglyakBolgoink** [MobileTribe/commit-template-idea-plugin#28]
Is it possible to sort the list?

![image](https://user-images.githubusercontent.com/2271337/124629858-26066480-de82-11eb-9a6a-6612acfe8fce.png)

**Comment by darrachequesne** [comment]
Yes, sure! Would you have the time to open a pull request for this feature?

**Comment by flyhelanman** [comment]
how can i delete the dropdown values ? thx

**Comment by flyhelanman** [comment]
> 

see this source code : https://github.com/MobileTribe/commit-template-idea-plugin/blob/master/src/com/leroymerlin/commit/GitLogQuery.java , use "git log cmd"  to show dropdown values ,so it can not delete。
